service/src/main/java/bio/terra/workspace/service/petserviceaccount/PetSaService.java

```diff
@@ -296,11 +296,12 @@ private Optional<Binding> findServiceAccountUserBinding(Policy saPolicy) {
    */
```

Comment:
```
  /**
   * Returns the pet service account of a provided user in a provided project if it exists in GCP. Returns empty Optional if pet SA doesn't exist, or if userEmail is a group instead of user.
   */
```